src/Loader.php

```diff
@@ -231,7 +232,7 @@ protected function sanitiseVariableValue($name, $value)
 
             // Unquoted values cannot contain whitespace
             if (preg_match('/\s+/', $value) > 0) {
-                throw new InvalidArgumentException('Dotenv values containing spaces must be surrounded by quotes.');
+                throw new SyntaxException();
```

Comment:
Yes - make it a parameter. That is the expected interface for the Exception class. I don't want to deviate from it.